contracts/standard/arbitration/ArbitrableDeposit.sol

```diff
@@ -107,10 +113,12 @@ contract ArbitrableDeposit is Arbitrable {
     function payArbitrationFeeByOwner() public payable onlyOwner{
         uint arbitrationCost = arbitrator.arbitrationCost(arbitratorExtraData);
         ownerFee += msg.value;
-        require(ownerFee == arbitrationCost); // Require that the total pay at least the arbitration cost.
-        require(status < Status.DisputeCreated); // Make sure a dispute has not been created yet.
+        // Require that the total pay at least the arbitration cost.
+        require(ownerFee == arbitrationCost, "Owner fee must equal arbitration cost.");
+        require(status < Status.DisputeCreated, "A dispute has already been raised."); // Make sure a dispute has not been created yet.
 
-        lastInteraction = now;
```

Comment:
There is nothing wrong with no which is an alias to block.timestamp, now is way more readable.